File: src/main/java/io/github/hdzitao/editstarters/startspringio/metadata/Env.java
package io.github.hdzitao.editstarters.startspringio.metadata;

import java.util.Map;

/**
 * configuration.env
 *
 * @version 3.2.0
 */
public class Env {
    private Platform platform;

    private Map<String, MetaDataBom> boms;
    private Map<String, MetadataRepository> repositories;

    public Platform getPlatform() {
        return platform;
    }

    public void setPlatform(Platform platform) {
        this.platform = platform;
    }

    public Map<String, MetaDataBom> getBoms() {
        return boms;
    }

    public void setBoms(Map<String, MetaDataBom> boms) {
        this.boms = boms;
    }

    public Map<String, MetadataRepository> getRepositories() {
        return repositories;
    }

    public void setRepositories(Map<String, MetadataRepository> repositories) {
        this.repositories = repositories;
    }
}

File: src/main/kotlin/io/github/hdzitao/editstarters/startspringio/metadata/Env.kt
package io.github.hdzitao.editstarters.startspringio.metadata

/**
 * configuration.env
 *
 * @version 3.2.0
 */
class Env(
    var platform: Platform? = null,
    var boms: Map<String, MetaDataBom>? = null,
    var repositories: Map<String, MetadataRepository>? = null
)
